mdpow/analysis/dihedral.py

```diff
@@ -78,7 +78,7 @@ def check_dihedral_inputs(selections):
         for group in selections:
             for k in group.keys():
                 if len(group[k]) != 4:
-                    msg = ''''Dihedral calculations require AtomGroups with
+                    msg = '''Dihedral calculations require AtomGroups with
                               only 4 atoms, %s selected''' % len(group)
```

Comment:
reformat this message so that it does not contain newline/space
```suggestion
                    msg = ("Dihedral calculations require AtomGroups with "
                           f"only 4 atoms, {len(group)} selected")
```
And we can use f-strings.